src/main/java/com/symbio/test/SeleniumPractice1/PageObj/WinePage.java

```diff
@@ -0,0 +1,41 @@
+package com.symbio.test.SeleniumPractice1.PageObj;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory;
+
+public class WinePage {
+	// wine
+	@FindBy(css = ".i2")
+	private WebElement chineseAndForeignWinesTab;
+
+	@FindBy(css = ".i-ext>a[href='http://list.jd.com/12259-12260-9437-0-0-0-0-0-0-0-1-1-1-1-1-72-4137-0.html']")
+	private WebElement wineLink;
+
+	@FindBy(css = ".gl-item>div[data-i='2']>.p-img")
+	private WebElement martell;
+
+	@FindBy(css = ".gl-item>div[data-i='1']>.p-img")
+	private WebElement hennessy;
+
```

Comment:
Specify variable name by adding variable type at the end of the name. E.g. hennessyItemLink